benchmark/main.cpp

```diff
@@ -965,13 +965,19 @@ static int benchmark( int argc, char *argv[], mbedtls_platform_context* ctx )
 
 int main(void) {
     mbedtls_platform_context platform_ctx;
-    if(mbedtls_platform_setup(&platform_ctx) != 0) {
-        return -1;
+    int exit_code = MBEDTLS_EXIT_FAILURE;
```

Comment:
Again, I think this function will always return MBEDTLS_EXIT_FAILURE, a failure code...

Reply:
No, the value will be overriden by platform_setup and benchmark(...).